core/sr-std/src/lib.rs

```diff
@@ -68,7 +68,7 @@ include!("../without_std.rs");
 pub mod prelude {
 	pub use crate::vec::Vec;
 	pub use crate::boxed::Box;
-	pub use crate::cmp::{Eq, PartialEq};
+	pub use crate::cmp::{Eq, PartialEq, Ordering};
```

Comment:
Why do we need this? (rust std also does not have it in prelude.)

Reply:
Good spotting, was a relict from a prior version. I removed it now.